tests/p_firefox/10-check_default_startpage.sh

```diff
@@ -11,9 +22,9 @@ if (t_GetArch firefox | grep -q 'x86_64')
   path='/usr/lib/firefox/defaults/preferences/all-redhat.js'
 fi
 
-count=$(grep -c www.centos.org $path)
+count=$(grep -c $website $path)
 
-if [ $count=2 ]
+if [ $count -eq 2 ]
```

Comment:
This is a fix to this test. This test is now failing because `www.centos.org` is not what the `firefox` package is bringing in their preferences: https://gitlab.com/redhat/centos-stream/rpms/firefox/-/blob/c9s/firefox-redhat-default-prefs.js#L17

Reply:
I just opened an issue for this one: RHEL-2409